lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -2485,7 +2485,8 @@ def quiver(self, *args,
                length=1, arrow_length_ratio=.3, pivot='tail', normalize=False,
                **kwargs):
         """
-        ax.quiver(X, Y, Z, U, V, W, /, length=1, arrow_length_ratio=.3, pivot='tail', normalize=False, **kwargs)
+        ax.quiver(X, Y, Z, U, V, W, /, length=1, arrow_length_ratio=.3, \
+pivot='tail', normalize=False, **kwargs)
```

Comment:
Can this be indented to align with the bracket in the line above?

Reply:
not really, that'll look awkward in the actual docstring (the problem is the same as always re: line continuations in rst).

Reply:
I think treating the first line of the docstring as a replacement `__signature__` is a sphinx thing, not an rst thing - it's possible it gets handled before the RST parser kicks in.